assets/queries/terraform/aws/eks_cluster_log_disabled/test/negative1.tf

```diff
@@ -0,0 +1,20 @@
+variable "cluster_name" {
+  default = "example"
+  type    = string
+}
+
+resource "aws_eks_cluster" "negative1" {
+  depends_on = [aws_cloudwatch_log_group.example]
+
+  enabled_cluster_log_types = ["api", "audit", "authenticator", "controllerManager", "scheduler"]
+  name                      = var.cluster_name
+
+  # ... other configuration ...
+}
+
+resource "aws_cloudwatch_log_group" "negative2" {
```

Comment:
split the samples

Reply:
The snippets are differents, but we don't need this part on negative example, will remove it